inception-pdf-editor/src/main/java/de/tudarmstadt/ukp/inception/pdfeditor/pdfanno/model/PdfExtractFile.java

```diff
@@ -260,4 +279,19 @@ public Offset getStringIndex(int aExtractIndex)
             return new Offset(sanitizedIndex, sanitizedIndex);
         }
     }
+
+    /**
+     * Get gegin and end offset of a page.
+     * @param page
```

Comment:
Either document parameter and return value with a proper description or remove the param/return stanzas.